tests/torch/quantization/test_quantizer_propagation_solver.py

```diff
@@ -897,13 +901,17 @@ def test_merged_qconfig_list_is_independent_of_branch_qconfig_list_order(self,
                                                 'target_branching_node_for_primary_quantizer',
                                                 'expected_status'))
 
+    InitNodeTestStruct = namedtuple('InitNodeTestStruct',
```

Comment:
Please don't use namedtuples.

Reply:
Just for my understanding, what significant advantages will I get from replacing a `namedtuple` with a `class`?

Reply:
You get better IDE completion, refactoring, type hints and the ability to do `isinstance` checks on corresponding objects with the specific class name instead of the generic `NamedTuple`. You lose in performance, which is not that relevant here; and even performance is better achieved using `__slots__` in regular classes rather than namedtuples.